shared/.flowconfig

```diff
@@ -44,6 +44,8 @@ sketchy-null-bool=off
 sketchy-null-string=off
 sketchy-null-mixed=off
 sketchy-null-number=off
+unclear-type=off
```

Comment:
this complains if you use any / Object /  function, which we do sometimes. kinda annoying